genie-agent/src/main/resources/genie-agent-defaults.yml

```diff
@@ -64,6 +64,7 @@ genie:
         max-files: 64000
         max-file-size: 8GB
         max-total-size: 16GB
+        check-remote-job-status: true
```

Comment:
Should we roll this out in two phases just to avoid any chance of a client sitting out there released if we have to roll the server back for some reason?

Reply:
Not sure what situation you're hinting at.

But we can roll this code out with this feature disabled by setting a fast property on the server.